nservicebus/pipeline/steps-stages-connectors_extra_core_[6.0,).partial.md

```diff
@@ -68,7 +68,7 @@ click Forwarding "/nservicebus/messaging/forwarding"
 
 ### Outgoing Pipeline Stages
 
- * Operation specific processing: There is a dedicated stage for each bus operation (e.g. Send, Publish, Subscribe, ...). Behaviors can use one of the following contexts: `IOutgoingSendContext`, `IOutgoingPublishContext`, `IOutgoingReplyContext`, `ISubscribeContext`, `IUnsubscribeContext`. Subscribe and Unsubscribe are not shown on the diagram below.
+ * Operation specific processing: There is a dedicated stage for each context operation (e.g. Send, Publish, Subscribe, ...). Behaviors can use one of the following contexts: `IOutgoingSendContext`, `IOutgoingPublishContext`, `IOutgoingReplyContext`, `ISubscribeContext`, `IUnsubscribeContext`. Subscribe and Unsubscribe are not shown on the diagram below.
```

Comment:
Context operation doesn't rhyme with me, how about just skipping it?

>There is a dedicated stage for each operation

or perhaps use: outgoing operation